Todo.Client/src/js/getTasks.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+import UITasks from './UITasks';
+
+
+const getTasks = async () => {
+  try {
+    const tasks = await axios.get('http://localhost:5000/api/tasks')
```

Comment:
Change 'http://localhost:5000' to const urlAPI

Reply:
good point, I was planning to change that and I forgot..
It would be also good to have this urlAPI variable defined in one place like separate config module, any ideas how to implement that?

Reply:
By using package https://www.npmjs.com/package/config
Add:
```suggestion
    const config = require('config');
    const urlAPI= config.get('ServerAPI.url');

```


and in config/default.json

```suggestion
{
  "ServerAPI": {
    "url": "http://localhost:5000"
  }
}
```